packages/paste-website/src/pages/changelog/index.mdx

```diff
@@ -4,7 +4,6 @@ title: Changelog
 
 import Changelog from '../../../CHANGELOG.md';
 
```

Comment:
Removed this blank line for a test when trying to commit with newer Prettier versions.